src/audio-worklet/free-queue/examples/simple-passthrough/coi-serviceworker.js

```diff
@@ -0,0 +1,115 @@
+/*! coi-serviceworker v0.1.6 - Guido Zuidhof, licensed under MIT */
```

Comment:
Can you explain why we need this? (with a top-level overview comment)

Reply:
Yes, I made duplicate of original file, as to not change anything or related licensing till am not sure about how to proceed with this file.

Reply:
No further action needed.